rma/models/stock_move.py

```diff
@@ -62,6 +62,7 @@ def _get_available_quantity(
             not lot_id
             and self.rma_line_id.lot_id
             and self.location_id.usage == "internal"
+            and self.rma_line_id.operation_id.out_force_same_lot
```

Comment:
I do see this PR a bit late but, are you sure this part is still necessary ?
Indeed, I see this PR now fill the `restrict_lot_id` in the procurement data, and then the module `stock_restrict_lot` should manage the reservations properly.

Reply:
In this case, what this is doing is being able to configure if the restrict lot is enabled or not by rma.operation and incoming or outgoing pickings. So this specific line is needed.

Reply:
Depending on the `out_force_same_lot` option of the `rma.operation`, the `restrict_lot_id` is or is not set on the `stock.move` : https://github.com/ForgeFlow/stock-rma/blob/16.0/rma/wizards/rma_make_picking.py#L142

Then, AFAIK, the `stock_restrict_lot` module take care of the reservation. Here, I feel like we do it twice...

Reply:
Shouldn't we delete the whole override ?
Because for me the problem is the lines 68 and 95. We do not need (I think) to force a lot in this module because it should be done already by `stock_restrict_lot`
If you juste delete L65 and L92, it is worse because it will force the RMA lot no matter if it is set on the `restrict_lot_id`.
See https://github.com/OCA/stock-logistics-workflow/blob/16.0/stock_restrict_lot/models/stock_move.py#L46

Reply:
Yes you are right, the two methods should be removed. @ArnauCForgeFlow